app/tests/spec/views/settings/emails.js

```diff
@@ -285,6 +292,48 @@ define(function (require, exports, module) {
           assert.equal(view.isPanelOpen(), false);
         });
       });
+
+      describe('can change email', () => {
+        const newEmail = 'secondary@email.com';
+        beforeEach(() => {
+          emails = [{
+            email: 'primary@email.com',
+            isPrimary: true,
+            verified: true
+          }, {
+            email: newEmail,
+            isPrimary: false,
+            verified: true
+          }];
+
+          return initView()
+            .then(function () {
+              // click events require the view to be in the DOM
+              $('#container').html(view.el);
+            });
+        });
+
+        it('can render', () => {
+          assert.equal(view.$('.email-address').length, 1);
+          assert.equal(view.$('.email-address .address').length, 1);
```

Comment:
You can use `assert.lengthOf` for these length checks, e.g., `assert.lengthOf(view.$('.email-address .address'), 1);`